src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import duke.command.AddCommand;
+import duke.command.Command;
+import duke.command.Commands;
+import duke.command.MarkCommand;
+import duke.command.UnmarkCommand;
+import duke.command.ListCommand;
+import duke.command.DeleteCommand;
+import duke.command.ByeCommand;
+import duke.command.FindCommand;
+import duke.exception.CommandInvalidException;
+
+public class Parser {
+
+    /**
+     * Returns validity whether the command given.
+     * If the command is valid, return true.
+     * If the command is invalid, return false.
+     *
+     * @param no given command.
+     * @return validity of the command.
+     *
+     */
```

Comment:
Is this comment necessary given that your javadocs already explains the purpose of this method?

Reply:
Not really, will erase it later. Thanks!